src/components/Item.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Item extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.handleObjectChange = this.handleObjectChange.bind(this);
```

Comment:
- [ ] [optional] I encountered the error below while running your app seems like you initialized a prop that you never used.
![image](https://user-images.githubusercontent.com/61408860/190173195-82b5fbcd-8308-4e92-b473-9b88097ba724.png)

Reply:
@danmainah Can you please help me with more details with regard to the first comment you gave up there with a screenshot from the terminal? How did you generate that in your local machine? Thanks